src/one_dragon/base/config/yaml_operator.py

```
@@ -6,8 +7,70 @@
+    cache_to_store = {}
+    for key, value in cached_yaml_data.items():
+        if _is_storeable_path(key):
+            cache_to_store[key] = value
+    if cache_to_store:
+        import json
+        with open('.cache_store.json', 'w', encoding='utf-8') as f:
+            json.dump(cache_to_store, f, ensure_ascii=False, indent=4)
+
+def walk_and_clear_cache():
+    """
+    遍历，若文件被修改则清除缓存
```

Ruff (0.13.3)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)